cloud-controller-manager/do/loadbalancers.go

```diff
@@ -360,20 +453,11 @@ func (l *loadBalancers) retrieveLoadBalancer(ctx context.Context, service *v1.Se
 	return l.lbByName(ctx, lbName)
 }
 
-func (l *loadBalancers) ensureLoadBalancerIDAnnot(service *v1.Service, lbID string) error {
-	if val := getLoadBalancerID(service); val == lbID {
-		return nil
-	}
-
-	// Make a copy so we don't mutate the shared informer cache from the cloud
-	// provider framework.
-	updated := service.DeepCopy()
-	if updated.ObjectMeta.Annotations == nil {
-		updated.ObjectMeta.Annotations = map[string]string{}
+func ensureServiceAnnotation(service *v1.Service, annotName, annotValue string) {
```

Comment:
`ensure*` usually indicates that a piece of logic changes something or makes sure that the change is already applied. `ensureServiceAnnotation` changed to simply recording a service annotation, so I'd opt to rename it to something like `recordServiceAnnotation`.